### PR TITLE
phishtank - add readme, fix typos, fix outputs types

### PR DESCRIPTION
**Pull request by bakatzir**
## Status
- [x] Ready

## Related Issues
fixes: https://github.com/demisto/etc/issues/26837

## Does it break backward compatibility?
   - [x] No

## Must have
- [x] Documentation

### CODE REVIEW

File: Packs/PhishTank/Integrations/README.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
+PhishTank is a free community site where anyone can submit, verify, track and share phishing data
+This integration was integrated and tested with version xx of PhishTank
```
Comment (Itay4) [inline Packs/PhishTank/Integrations/README.md:2]: update version

File: Packs/PhishTank/Integrations/README.md [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+| DBotScore.Type | unknown | The type of the indicator | 
+| DBotScore.Vendor | unknown | Vendor used to calculate the score | 
+| DBotScore.Score | unknown | The actual score | 
+
+
+#### Command Example
+```!url url=https://www.demisto.com```
+
+#### Human Readable Output
+<br>PhishTank Database - URL Query </br>
```
Comment (Itay4) [inline Packs/PhishTank/Integrations/README.md:51]: perhaps show example of phishing url ?

File: Packs/PhishTank/Integrations/README.md [hunk 3]
```diff
@@ -0,0 +1,106 @@
...
+Reload PhishTank database
+
+
+#### Base Command
+
+`phishtank-reload`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
```
Comment (Itay4) [inline Packs/PhishTank/Integrations/README.md:66]: i think this need to replaced with something like `This command has no inputs`

File: Packs/PhishTank/Integrations/README.md [hunk 4]
```diff
@@ -0,0 +1,106 @@
...
+***
+Show PhishTank database status
+
+
+#### Base Command
+
+`phishtank-status`
+#### Input
+
+| **Argument Name** | **Description** | **Required** |
```
Comment (Itay4) [inline Packs/PhishTank/Integrations/README.md:91]: same here

File: Packs/PhishTank/Integrations/README.md [hunk 5]
```diff
@@ -0,0 +1,106 @@
...
+
+| **Parameter** | **Description** | **Required** |
+| --- | --- | --- |
+| proxy | Use system proxy settings | False |
+| insecure | Trust any certificate \(not secure\) | False |
+| fetchIntervalHours | Database refresh interval \(hours\) | False |
+
+4. Click **Test** to validate the URLs, token, and connection.
+## Commands
+You can execute these commands from the Demisto CLI, as part of an automation, or in a playbook.
```
Comment (Itay4) [inline Packs/PhishTank/Integrations/README.md:17]:
```suggestion
You can execute these commands from the Cortex XSOAR CLI, as part of an automation, or in a playbook.
```

File: Packs/PhishTank/Integrations/README.md [hunk 6]
```diff
@@ -0,0 +1,106 @@
...
+| **Argument Name** | **Description** | **Required** |
+| --- | --- | --- |
+| url | URL to check | Required | 
+
+
+#### Context Output
+
+| **Path** | **Type** | **Description** |
+| --- | --- | --- |
+| URL.Data | unknown | Bad URLs found |
```
Comment (Itay4) [inline Packs/PhishTank/Integrations/README.md:38]: lets update the types

File: Packs/PhishTank/Integrations/README.md (hunk 1 above)
Comment (bakatzir) [reply Packs/PhishTank/Integrations/README.md:2]: web service, no version. i'll update.

File: Packs/PhishTank/Integrations/README.md (hunk 2 above)
Comment (bakatzir) [reply Packs/PhishTank/Integrations/README.md]: done. escaped and added to pack-ignore.

**Pull request merged**